modules/deploy/main.tf

```diff
@@ -67,6 +67,7 @@ if [[ $STATUS == "Succeeded" ]]; then
     echo "Deployment succeeded."
 else
     echo "Deployment failed!"
+    exit 1
```

Comment:
Good point, but maybe `deploy get-deployment` (line 73) should be also executed before exiting to see the error message? For example, move line 73 before the `if` statement on line 66.

What do you think?

Reply:
Yes, that makes sense. I switched the order for the case where `wait_deployment_completion` is false so the pattern is consistent.